README.md

```diff
@@ -257,10 +257,46 @@ If you enable ProGuard, then add this rules in your ProGuard file.
 ```
 
 
+## Note on TLS 1.2
+PCI-DSS standard requires the service to communicate in *TLS 1.2* or higher using strong encryption suite. This means that every clients must connect to Omise service with those valid suites. However TLS 1.2 support in Android is vary depends on the Android OS.
+
```

Comment:
Please add `Please follow the following instruction to add support for TLS 1.2 in your app` before the new heading

Sorry I missed again. However you can merge this PR after you add it here :)

Reply:
@pitiphong-p Do you mean the end of this paragraph?

Reply:
Correct. Right where I mark for this comment